src/components/atoms/Breadcrumbs/Breadcrumbs.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import HomeIcon from '@mui/icons-material/Home';
+import Link from '@mui/material/Link';
+import { Meta, StoryFn } from '@storybook/react';
+import { Box, Breadcrumbs, BreadcrumbsProps, Button } from 'src/components';
+
+export default {
+  title: 'Components/Breadcrumbs',
+  component: Breadcrumbs,
+} as Meta;
+
+const Template: StoryFn<BreadcrumbsProps> = (args) => <Breadcrumbs {...args} />;
```

Comment:
Suggestion: Emphasize more the presence of children
```suggestion
const Template: StoryFn<typeof Breadcrumbs> = ({ children, ...args }) => (
  <Breadcrumbs {...args}>{children}</Breadcrumbs>
);
```